### PR TITLE
Clarify use of Sonos favorites sensor in service calls

### PR DESCRIPTION
**Pull request by dreed47**
## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Review by jjlawren — DISMISSED** [review]
Thanks, this is a good clarification. 👍

File: source/_integrations/sonos.markdown [hunk 1]
```diff
@@ -69,7 +69,18 @@ The microphone can only be enabled/disabled from physical buttons on the Sonos d
...
 
-If using the provided `media_content_id` with the `media_player.play_media` service, the `media_content_type` must be set to "favorite_item_id".
+When calling the `media_player.play_media` service, the `media_content_type` must be set to "favorite_item_id" and the `media_content_id` must be set to just the key portion of the favorite item. 
+
+Example service call:
+
+```yaml
+service: media_player.play_media
+target:
+  entity_id: all
```
Comment (frenck) [inline source/_integrations/sonos.markdown]: This targets also non-Sonos entities. Let's make the example more specific.

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_integrations/sonos.markdown (hunk 1 above)
Comment (dreed47) [reply source/_integrations/sonos.markdown]: is there a way to target just all Sonos?

Comment (frenck) [reply source/_integrations/sonos.markdown]: No, there is no way. Please just use a made-up entity as an example instead.

**Review by frenck — APPROVED** [review]
Thanks, @dreed47 👍

**Pull request merged**